src/components/ProcessDocumentScreen/ProcessDocumentContent/ProcessDocumentContent.stories.tsx

```diff
@@ -0,0 +1,130 @@
+import { FunctionComponent } from "react";
+import { ProcessDocumentContent } from ".";
+import { QueueState, QueueType } from "../../../constants/QueueState";
+import { WrappedDocument } from "../../../types";
+
+export default {
+  title: "ProcessDocumentScreen/ProcessDocumentContent",
+  component: ProcessDocumentContent,
+  parameters: {
+    componentSubtitle: "ProcessDocumentContent.",
+  },
+};
+
+const mockDocs = [
+  {
+    type: "VERIFIABLE_DOCUMENT",
+    contractAddress: "",
+    fileName: "COO-TEST-1",
+    payload: {},
+    rawDocument: {},
+    wrappedDocument: { data: "TEST TEST" },
+  },
+] as WrappedDocument[];
+
+export const issueSuccess: FunctionComponent = () => {
+  return (
+    <ProcessDocumentContent
+      network={"local"}
+      queueState={QueueState.CONFIRMED}
+      queueType={QueueType.ISSUE}
+      failedDocuments={[]}
+      pendingDocuments={[]}
+      successDocuments={mockDocs}
+      fileName={"file-name-test"}
+      downloadErrorName={""}
+      downloadErrorLink={""}
+      downloadAllFn={() => {}}
+      processAnotherDocumentFn={() => {}}
+    />
+  );
+};
+
+export const issueFailure: FunctionComponent = () => {
+  return (
+    <ProcessDocumentContent
+      network={"local"}
+      queueState={QueueState.CONFIRMED}
+      queueType={QueueType.ISSUE}
+      failedDocuments={mockDocs}
+      pendingDocuments={[]}
+      successDocuments={[]}
+      fileName={"file-name-test"}
+      downloadErrorName={""}
+      downloadErrorLink={""}
+      downloadAllFn={() => {}}
+      processAnotherDocumentFn={() => {}}
+    />
+  );
+};
+
+export const issueSuccessAndFailure: FunctionComponent = () => {
+  return (
+    <ProcessDocumentContent
+      network={"local"}
+      queueState={QueueState.CONFIRMED}
+      queueType={QueueType.ISSUE}
+      failedDocuments={mockDocs}
+      pendingDocuments={[]}
+      successDocuments={mockDocs}
+      fileName={"file-name-test"}
+      downloadErrorName={""}
+      downloadErrorLink={""}
+      downloadAllFn={() => {}}
+      processAnotherDocumentFn={() => {}}
+    />
+  );
+};
+
+export const issueError: FunctionComponent = () => {
+  return (
+    <ProcessDocumentContent
+      network={"local"}
+      queueState={QueueState.ERROR}
+      queueType={QueueType.ISSUE}
+      failedDocuments={[]}
+      pendingDocuments={[]}
+      successDocuments={[]}
+      fileName={"file-name-test"}
+      downloadErrorName={""}
+      downloadErrorLink={""}
+      downloadAllFn={() => {}}
+      processAnotherDocumentFn={() => {}}
+    />
+  );
+};
+export const revokeSuccess: FunctionComponent = () => {
+  return (
```

Comment:
the file name only works here in storybook but not the rest of the cases, something to do with this https://github.com/TradeTrust/document-creator-website/blob/21fba047146c40e3f0ac2ee56ba7ea3dd71154bb/src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx#L31